src/main/java/cms/gov/madie/measure/services/BundleService.java

```diff
@@ -0,0 +1,60 @@
+package cms.gov.madie.measure.services;
+
+import cms.gov.madie.measure.exceptions.BundleOperationException;
+import cms.gov.madie.measure.exceptions.CqlElmTranslationErrorException;
+import cms.gov.madie.measure.exceptions.CqlElmTranslationServiceException;
+import cms.gov.madie.measure.exceptions.InvalidResourceBundleStateException;
+import com.nimbusds.oauth2.sdk.util.StringUtils;
```

Comment:
Prefer Apache Common's StringUtils over nimbusds just to maintain consistency and develop expectations.

Reply:
Fixed